packages/massa-web3/src/experimental/basicElements/operationManager.ts

```diff
@@ -110,17 +117,6 @@ export class OperationManager {
         operation = operation as RollOperation
         components.push(unsigned.encode(operation.amount))
         break
-      case OperationType.CallSmartContractFunction:
-        // @see https://docs.massa.net/docs/learn/operation-format-execution#callsc-operation-payload
-        operation = operation as CallOperation
-        components.push(unsigned.encode(operation.maxGas))
-        components.push(unsigned.encode(operation.coins))
-        components.push(Address.fromString(operation.address).toBytes())
-        components.push(varint.encode(operation.func.length))
-        components.push(Buffer.from(operation.func))
-        components.push(varint.encode(operation.parameter.length))
-        components.push(operation.parameter)
-        break
```

Comment:
I assume we don't need this code thanks to Ben's refacto on callSC ?

Reply:
My bad, I'll fix it.